itou/users/management/commands/extract_c2_users.py

```diff
@@ -82,6 +82,7 @@ def get_basic_row(self, membership, org):
             "DateRattachement": membership.created_at.date(),
             "Département": DEPARTMENTS[org.department] if org.department else None,
             "Région": org.region,
+            "Organisation": org.name,
```

Comment:
Ici j'utiliserais plutôt `display_name` car c'est ce qu'on envois au pilotage : 
https://github.com/gip-inclusion/les-emplois/blob/2c8481a107612d3d01743ef343563935d14aa098/itou/metabase/tables/organizations.py#L74
https://github.com/gip-inclusion/les-emplois/blob/2c8481a107612d3d01743ef343563935d14aa098/itou/metabase/tables/companies.py#L30
```suggestion
            "Organisation": org.display_name,
```

Reply:
Bien vu !